### PR TITLE
1190 - working on the modal for running the report

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/src/presenter/actions/setupTrialYearsAction.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { state } from 'cerebral';
+
+/**
+ * sets the state.trialYears
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.store the cerebral store used for setting state.users
```
Comment (matthopson) [inline web-client/src/presenter/actions/setupTrialYearsAction.js]: maybe just remove `...used for setting state.users`

File: web-client/src/views/TrialSessionPlanningModal.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import { sequences, state } from 'cerebral';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TrialSessionPlanningModal = connect(
+  {
+    cancelSequence: sequences.clearModalSequence,
+    confirmSequence: sequences.runTrialSessionPlanningReportSequence,
+    modal: state.modal,
+    trialYears: state.trialYears,
```
Comment (rachaelparris) [inline web-client/src/views/TrialSessionPlanningModal.jsx]: Should the trialYears also be on state.modal so they're cleared when the modal state is cleared? Or is this not a big deal?

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/utilities/DateHandler.js [hunk 3]
```diff
@@ -58,11 +58,16 @@ const formatNow = formatStr => {
   return module.exports.formatDateString(now, formatStr);
 };
 
+const getCurrentYear = () => {
+  return new Date().getFullYear();
```
Comment (matthopson) [inline shared/src/business/utilities/DateHandler.js]: Could use `formatNow('YYYY')` here.

**Review by matthopson — APPROVED** [review]
Couple small nits, otherwise 👍

File: web-client/src/views/TrialSessionPlanningModal.jsx (hunk 2 above)
Comment (codyseibert) [reply web-client/src/views/TrialSessionPlanningModal.jsx]: yes, they probably should

**Pull request merged**